sparkle/src/components/markdown/ExtendedMarkdown.tsx

```diff
@@ -0,0 +1,558 @@
+/* eslint-disable @typescript-eslint/no-explicit-any */
```

Comment:
This is because of unist-util-visit - this rule is disabled in front, it requires a lot of changes to have specific typings